### PR TITLE
Remove pause annotation

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/15502
Instead of https://github.com/giantswarm/app-operator/pull/689

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/app/mutate_app.go
Comment (kopiczko) [inline pkg/app/mutate_app.go:257]: We can remove `mutateManagementClusterApp` func instead. Those apps should not be reconciled by config-controller anymore since your release-operator release.

Comment (kopiczko) [reply pkg/app/mutate_app.go:257]: Scratch that we need to update `opsctl` frist

**Review by kopiczko — APPROVED** [review]

**Comment by corest** [comment]
Ok, waiting for @giantswarm/team-batman-engineers approval and merging then

**Review by rossf7 — APPROVED** [review]

**Pull request merged**
